### PR TITLE
Handle OTEL_EXPORTER_OTLP_SPAN_INSECURE 

### LINKED ISSUE #89: Handle OTEL_EXPORTER_OTLP_SPAN_INSECURE
We need to support the old `OTEL_EXPORTER_OTLP_SPAN_INSECURE` (and mark it as **deprecated**), in case the user tried to set it up still.

We should do as follows:

1) If `OTEL_EXPORTER_OTLP_SPAN_INSECURE` is defined, save its value.
2) If `OTEL_EXPORTER_OTLP_SPAN_ENDPOINT` already contains either `http://` or `https://`, discard value from 1), else append `http://` or `https://` respectively.

### PR DESCRIPTION
**Pull request by malafeev**
fix #89

### CODE REVIEW

File: common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java [hunk 1]
```diff
@@ -41,9 +45,12 @@ public static void setSystemProperties(Configuration configuration, boolean isAg
 
     if (!configuration.spanEndpoint.toLowerCase().startsWith("http://")
         && !configuration.spanEndpoint.toLowerCase().startsWith("https://")) {
-      String msg = "Invalid configuration: span endpoint should start with http:// or https://.";
-      logger.severe(msg);
-      throw new IllegalStateException(msg);
+
```
Comment (carlosalberto) [inline common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java:48]: Can we add a comment here to point out this is something done to keep backwards compatibility?

File: common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java [hunk 2]
```diff
@@ -41,9 +45,12 @@ public static void setSystemProperties(Configuration configuration, boolean isAg
...
+      if (configuration.insecureTransport) {
+        configuration.spanEndpoint = "http://" + configuration.spanEndpoint;
```
Comment (carlosalberto) [inline common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java:50]: Let's do a pass in a follow-up PR (in the future), saving these constants (`http`, `https`, etc) in a `Constants.java` file ;)

File: common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java [hunk 3]
```diff
@@ -218,6 +226,11 @@ public static String getPropagator() {
     return getProperty(OTEL_PROPAGATORS, DEFAULT_PROPAGATOR);
   }
 
+  public static boolean useInsecureTransport() {
```
Comment (carlosalberto) [inline common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java:229]: Can we make this method private? I don't think this needs to be public (if **yes**, then let's mark this as deprecated).

Comment (malafeev) [reply common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java:229]: no, it's used in launcher (manual configuration)

File: common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java (hunk 1 above)
Comment (malafeev) [reply common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java:48]: added

File: common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java (hunk 2 above)
Comment (malafeev) [reply common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java:51]: ok

**Review by carlosalberto — APPROVED** [review]

**Comment by carlosalberto** [comment]
Let's release this ;)

**Pull request merged**